feat(cmf): add bootstrap option to not render the app (#3090)
Talend/ui: packages/cmf/src/bootstrap.md

| attribute | default value | description                                        |
| --------- | ------------- | -------------------------------------------------- |
| reportURL | undefined     | the error where to post. For example '/api/errors' |

## The case of angular

> If you are using ngreact and you want to leverage cmf here is how you should call bootstrap:
> 
> ```javascript
> import cmf from '@talend/react-cmf';
> 
> import containersModule from '@talend/react-containers';
> import components from './components';
> import saga from './saga';
> import sagas from './sagas';
> 
> const setup = cmf.boostrap({
> 	render: false,
> 	components,
> 	saga,
> 	sagas,
> 	modules: [containersModule],
> });
> ```
> 
> | attributes      | description                              |
> | --------------- | ---------------------------------------- |
> | store           | redux store instance                     |
> | saga.run        | if you want to start the saga middleware |
> | saga.middleware | the instance of saga middleware          |
> | App             | cmf App component to render a CMF App    |
> | cmfModule       | the merged options from all cmfModules   |

Reviewer: ```suggestion
If you are using ngreact and you want to leverage cmf you must set the `render` option to `false`:

```javascript
import cmf from '@talend/react-cmf';

import containersModule from '@talend/react-containers';
import components from './components';
import saga from './saga';
import sagas from './sagas';

const setup = cmf.boostrap({
	render: false,
	components,
	saga,
	sagas,
	modules: [containersModule],
});
```

The result from `cmf.boootstrap()` contains those attributes:

| attributes      | description                              |
| --------------- | ---------------------------------------- |
| store           | redux store instance                     |
| saga.run        | if you want to start the saga middleware |
| saga.middleware | the instance of saga middleware          |
| App             | cmf App component to render a CMF App    |
| cmfModule       | the merged options from all cmfModules   |
```
